minio/minio
# site replication: Add support for site level resync

## Description


## Motivation and Context
Provide ability to resync data if a participant in cluster replication loses its data

## How to test this PR?
Needs https://github.com/minio/pkg/pull/40 and https://github.com/minio/madmin-go/pull/129 to build 

mc PR - https://github.com/minio/mc/pull/4269
```
   mc admin replicate add minio1 minio2
   // create some buckets
   // remove some or all bucket data from minio2 and restart clusters
   mc admin replicate resync start minio minio2 // should resync the content
```

## Types of changes

## Checklist:
